spec/models/book_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+require 'faker'
+
+describe Book do
```

Comment:
Add validations tests:
```
it { is_expected.to validate_presence_of(:title) }
```

Reply:
Done. I has implemented shoulda library for this validation.